uaclient/entitlements/ros.py

```diff
@@ -0,0 +1,11 @@
+from uaclient.entitlements import repo
+
+
+class ROSESMEntitlement(repo.RepoEntitlement):
+    help_doc_url = ""
```

Comment:
@mitechie should this doc URL link  be https://ubuntu.com/robotics/ros-esm and should we work with @lechsandecki to make sure design folks are on board with making this the documentation landing point for ROS sort of like we have https://ubuntu.com/security/esm which onboards to setting up Ubuntu Advantage?

Reply:
I think the URL https://ubuntu.com/robotics/ros-esm is ok and works. We can see if PMs (and this falls more under Gabriel as the ROS PM) want to change it once they play with it in beta form.